<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergeLayers</class>
 <widget class="QDialog" name="MergeLayers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Merge Layers</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Merge layers</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labFroLayer">
          <property name="text">
           <string>Merge Layer...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbFromLayer"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labToLayer">
          <property name="text">
           <string>...to Layer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbToLayer"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cbDeleteLayer">
        <property name="text">
         <string>Delete layer after merge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnMerge">
          <property name="text">
           <string>Merge Layers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClose">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
